graph/client/src/main.tsx

```diff
@@ -9,30 +9,58 @@ import { inspect } from '@xstate/inspect';
 import { App } from './app/app';
 import { ExternalApiImpl } from './app/external-api-impl';
 import { render } from 'preact';
+import { ErrorToastUI, ExpandedTargetsProvider } from '@nx/graph/shared';
+import { ProjectDetails } from '@nx/graph-internal/ui-project-details';
+import { ErrorPage } from './app/ui-components/error-page';
 
-if (window.useXstateInspect === true) {
-  inspect({
-    url: 'https://stately.ai/viz?inspect',
-    iframe: false, // open in new window
-  });
-}
-
-window.externalApi = new ExternalApiImpl();
-const container = document.getElementById('app');
+if (window.__NX_RENDER_GRAPH__ === false) {
```

Comment:
Ideally, we'd have `renderPDV`, `renderError` and `renderGraph` functions exposed. But for older Nx versions, obviously we need to support the old way.

Can we check on the Console side, whether we're on 19.8, and then call `renderGraph` rather than assuming the main file will call `render` automatically?